profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module

```diff
@@ -298,26 +296,27 @@ function _nexteuropa_multilingual_alias_node_form_alter(&$form, &$form_state) {
  *   - alias: The the LANGUAGE_NONE alias.
  *   - pathauto: The flag indicating if pathauto is activated for the node.
  */
-function _nexteuropa_multilingual_get_und_entity_alias_info($entity, $entity_type) {
+function _nexteuropa_multilingual_get_und_entity_alias_info($entity_type, $entity) {
```

Comment:
Is there a particular reason why you switched the order of the arguments? I would leave it like it was, thus providing a patch for the bug and nothing more, reducing the diff and limiting risks.